Solutions/RubrikSecurityCloud/Playbooks/RubrikAnomalyIncidentResponse/azuredeploy.json

```diff
@@ -7,7 +7,13 @@
         "prerequisites":  [
             "1. The Rubrik Security Cloud data connector should be configured to send appropriate events to Microsoft Sentinel.",
             "2. The Rubrik Security Cloud solution should be configured to [connect to Rubrik Security Cloud API end points using a Service Account](https://docs.rubrik.com/en-us/saas/saas/polaris_api_access_with_service_accounts.html), the service account should be assigned a role that includes the relevant privileges necessary to perform the desired operations (see [Roles and Permissions](https://docs.rubrik.com/en-us/saas/saas/common/roles_and_permissions.html) in the Rubrik Security Cloud user guide).",
-            "3. Obtain Teams group id and channel id."
+            "3. Obtain Teams group id and channel id.",
+            "4. Make sure that below mentioned playbooks are deployed before deploying RubrikAnomalyIncidentResponse playbook:
+                - RubrikDataObjectDiscovery
+                - RubrikAnomalyAnalysis
+                - RubrikFilesetRansomwareDiscovery
+                - RubrikRansomwareDiscoveryAndFileRecovery
+                - RubrikRansomwareDiscoveryAndVMRecovery"
```

Comment:
This string (point 4) is not formatted correctly, causing the ARM template failure.